src/fu-util.c

```diff
@@ -1953,6 +1953,16 @@ fu_util_switch_branch_warning (FuUtilPrivate *priv,
 	return TRUE;
 }
 
+static const gchar *
+fu_util_get_branch_fallback (const gchar *nullable_branch)
+{
+	if (nullable_branch == NULL) {
+		/* TRANSLATORS: this is the default branch name when unset */
+		return _("default");
```

Comment:
this could be a different behavior of this will be different than `fwupdtool`.  I would say just pull the translation into the engine too in case it's launched from fwupdtool

Reply:
We can't push translated content over the D-Bus interface; we could push the fu_util_get_branch_fallback() into fu-util-common.c tho...

Reply:
Yeah I think that works

Reply:
Done, it's nicer as an API too.